hedera-node/hedera-mono-service/src/test/resources/bootstrap/standard.properties

```diff
@@ -110,6 +110,7 @@ hedera.transaction.maxMemoUtf8Bytes=100
 hedera.transaction.maxValidDuration=180
 hedera.transaction.minValidDuration=15
 hedera.transaction.minValidityBufferSecs=10
+hedera.transaction.eip2930.enabled=true
```

Comment:
Is it ok that `standard.properties` doesn't match `hedera-mono-service/src/main/.../resources/bootstrap.properties`?  Is the _only_ reason for `standard.properties` to help the tests of global dynamic properties?  (@Neeharika-Sompalli  - this might be something you can answer for me, thanks!)

Reply:
is `hedera.transaction` the correct prefix for this property?  That prefix implies to me that it is applicable to all Hedera transactions but maybe it is only meaningful in the context of contracts/ethereum transactions?

Reply:
@david-bakin-sl  `standard.properties` are used for tests as far as I know. But `hedera-mono-service/src/main/.../resources/bootstrap.properties` are the default properties for node.